charts/incubator/ghostfolio/templates/_configmap.tpl

```diff
@@ -0,0 +1,27 @@
+{{/* Define the configmap */}}
+{{- define "ghostfolio.configmap.paths" -}}
+enabled: true
+data:
+  ACCESS_TOKEN_SALT: "/secrets/ACCESS_TOKEN_SALT"
+  JWT_SECRET_KEY: "/secrets/JWT_SECRET_KEY"
+  DATABASE_URL: "/secrets/DATABASE_URL"
+  REDIS_PASSWORD: "/secrets/REDIS_PASSWORD"
+{{- end -}}
+
+{{- define "ghostfolio.configmap.configfile" -}}
+enabled: true
+data:
+  .env: |
+    COMPOSE_PROJECT_NAME=ghostfolio
+
+    # CACHE
+    REDIS_HOST={{ .Values.redis.creds.plain }}
+    {{- with $redis := .Values.redisProvider }}
```

Comment:
What is this `with`? There is no `redisProvider` in values.
Even if it was, it won't be optional